xml_inscriptions/transcribed/MI.AA.UM.KM.L.819and857.xml.bak

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?><?oxygen RNGSchema="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" type="xml"?><TEI xmlns:xi="http://www.w3.org/2001/XInclude" xmlns="http://www.tei-c.org/ns/1.0">
```

Comment:
Pretty sure this file is extraneous; I'll come back to it and check after merging.